syntax = "proto3";

package common;

message Track {
    string spotify_track_id = 1;
    string track_name = 2;
    string artist = 3;
    string genre = 4;
    float tempo = 5;
    float energy = 6;
    float valence = 7;
    float danceability = 8;
  }